PRS5001-6000/PRS5276Helena.xml

```diff
@@ -32,26 +32,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-04-28">batch updated according to requirements and issues.</change>
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change who="MKr" when="2021-12-21"></change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="2">
-               <persName>
-                  
-                  Helena
-               </persName>
-               <occupation> emp. </occupation>
+            <person sex="2" sameAs="wd:Q170164">
+               <persName xml:lang="en" xml:id="n1">Helena</persName>
+               <persName xml:lang="gez" corresp="#n1">እሌኒ፡</persName>
+               <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleni</persName>
+               <occupation type="political">empress</occupation>
+               <birth notBefore="0248" notAfter="0250">248-250</birth>
+               <death when="0328">328</death>
+               <nationality type="ByzantineEmpire"></nationality>
             </person>
+            <listRelation>
+              <relation name="snap:MotherOf" active="PRS5276Helena" passive="PRS3175Constant"/>
+            </listRelation>
```

Comment:
maybe also add subject_of relation to NAR0145SenHelena
btw, could LIT6171HolyCross be possibly doublet to LIT1764Legend ?

Reply:
Nobody has ever explained to me what NAR is so I'm not using it:( As for the doublet, we had this discussion https://github.com/BetaMasaheft/Documentation/issues/1619 There is article by Witold